worked on save and load

Marchybear opened the pull request:
kinda working but not done (have to figure out how to work with arraylists), theres more user prompts in main now, watch out!

Marchybear committed:
worked on save and load

ambersjliu commented on battleship/SaveLoad.java:
```diff
@@ -0,0 +1,88 @@
+
+    public void Save(int enemyMisses, int enemyHits, int ourHits, int ourMisses, ArrayList<Coordinate> pastShots){
+        try{
+            FileOutputStream saveFile = new FileOutputStream("SaveFile.sav");
```
not sure about using sav file lmao, let's just use .txt
and in that case we can use bufferedwriter/filewriter (?)

Marchybear committed:
Save and Load almost done

Marchybear commented:
pretty much most of the things the needs to be saved works, ill figure out ships tmr

also add timer tmr hopefully 

oh and ig if someone save mid target mode ill have to save the ai target stuff, right?

Marchybear committed:
save and load done!

the try and catch while loop (for file not found), isnt looping for some reason... other than that its godd :))
